### PR TITLE
Task/markup improvements

### PR DESCRIPTION
**Pull request by mmvsha**

### CODE REVIEW

File: src/store.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { hashHistory } from 'react-router'
 import { createStore, applyMiddleware, compose } from 'redux'
 import thunk from 'redux-thunk'
+import logger from 'redux-logger'
```
Comment (andre-meyer) [inline src/store.js]: Why do you use `redux-logger`? The redux dev tools are much more in depth: https://github.com/gnosis/gnosis-management/pull/65/files#diff-3bf35cd41da51153dcca3003656f7c67R35

**Comment by andre-meyer** [comment]
In the future, when you want to change styling issues such as surrounding parentheses in short-ifs, e.g. `prop={( foo ? 'bar' : 'baz' )}` make sure they're in the linting rules, if not, they're probably not going to get respected and happen again and again. 

After such changes, propose PRs for fixing all instances of those styling issues at once, so these changes are not too spread out. Thank you!

**Comment by mmvsha** [comment]
@andre-meyer need some review again! Forgot to add WIP label, sorry

File: src/components/MarketResolveForm/index.js [hunk 2]
```diff
@@ -41,23 +36,34 @@ class MarketResolveForm extends Component {
...
+          Resolve Oracle
+        </button>
       </form>
     )
   }
 
   renderResolveCategorical() {
     const { handleSubmit, market: { eventDescription: { outcomes } } } = this.props
-
+    console.log(this.props);
```
Comment (andre-meyer) [inline src/components/MarketResolveForm/index.js]: remove please

File: src/components/MarketResolveForm/index.js [hunk 3]
```diff
@@ -66,16 +72,21 @@ class MarketResolveForm extends Component {
     const { submitting, market: { event: { type }, oracle: { isOutcomeSet } } } = this.props
 
     if (submitting) {
+      console.log('SUBMITTING')
```
Comment (andre-meyer) [inline src/components/MarketResolveForm/index.js]: same

File: src/components/MarketResolveForm/index.js [hunk 4]
```diff
@@ -66,16 +72,21 @@ class MarketResolveForm extends Component {
...
       return <span>Resolving Oracle...</span>
     }
 
     if (isOutcomeSet) {
+      console.log('isOutComeSet')
```
Comment (andre-meyer) [inline src/components/MarketResolveForm/index.js]: same

**Comment by andre-meyer** [comment]
Is the server configured to handle browserHistory as a router? e.g. `management-beta.gnosis.pm/markets/list` currently returns a 404

**Comment by mmvsha** [comment]
@andre-meyer we don't have our server configured yet. Currently, webpack-dev-server is running on our server and it serves all the routes so it won't be a problem

**Pull request merged**